Makefile

```diff
@@ -390,6 +405,10 @@ docker-push:
 		docker push $(DOCKER_NAMESPACE)/jaeger-$$component ; \
 	done
 
+.PHONY: dockerx-push
+dockerx-push: setup-buildx-builder
```

Comment:
Is this a WIP?  The all-in-one build is only doing amd64, I assume because BUILDX_PUSH=false by default. Wouldn't we want x-platform build to be the default, rather than having a separate target?

Reply:
`all-in-one` is not in `make docker-push`, so i haven't add it in `make dockerx-push`